README.md

```diff
@@ -46,6 +46,8 @@ the application is being run. This is best practice as described by
 | `ES_HOST`                          | host name of Elasticsearch server                                        |                         | yes        |
 | `ES_INDEX`                         | Elasticsearch configuration to read, currently only 'profiles' available | profiles                |            |
 | `ES_PORT`                          | port of Elasticsearch server                                             | 27017                   |            |
+| `ES_REPLICAS`                      | Number of replica for the index                                          | 2                       |            |
```

Comment:
Why default to 2 copies of the data rather than 1? This will mean there will be 15 shards in total. Does the hardware configuration available mean that having the additional copy of the data will provide more resiliency?